code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -340,6 +340,8 @@
 			employmentCabinet.addFile(employee)
 
 /mob/dead/new_player/proc/LateChoices()
+	var/password = ""
+	var/password_cost = 500
```

Comment:
this can be made into a define at the top of the file
```
#define PASSWORD_COIN_COST 500
```

and then at the bottom:
```
#undef PASSWORD_COIN_COST
```
we're not going to be changing this around, so it's better off this way

Reply:
this is no longer a static variable